src/binary-reader-objdump.cc

```diff
@@ -296,6 +297,16 @@ Result BinaryReaderObjdumpDisassemble::OnOpcodeBare() {
   return Result::Ok;
 }
 
+Result BinaryReaderObjdumpDisassemble::OnOpcodeIndex(Index value) {
```

Comment:
Minor question. Since the calling context should know if the index corresponds to a call, why not have two different functions "OnOpcodeIndex()" and "OnOpcodeCallIndex()"?

In this view, you wouldn't need to check the current_opcode, and OnOpcodeIndex() would not need to be conditionalized. However, OnOpcodeCallIndex still would, since there may not be a function name.